src/org/usfirst/frc/team2706/robot/subsystems/DriveTrain.java

```diff
@@ -63,6 +71,11 @@ public DriveTrain() {
 		// @TODO: Use RobotMap values
 		rangefinder = new AnalogInput(6);
 		gyro = new AnalogGyro(1);
+		
+		gyroPIDSource = new GyroPIDSource(this);
+		encoderPIDSource = new EncoderPIDSource(this);
+		
+		drivePIDOutput = new DrivePIDOutput(this);
 
 		// Let's show everything on the LiveWindow
 		LiveWindow.addActuator("Drive Train", "Front_Left Motor", (Talon) front_left_motor);
```

Comment:
The (talon) typecast will cause problems when we switch to victors. We should avoid typecast as much as possible.